README.adoc

```diff
@@ -171,6 +231,46 @@ curl http://$(minikube ip):32000/inventory/systems/system-service
 ```
 --
 
+endif::[]
+
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+In this execise, you need to set up port forwarding to access the services.
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Run the following commands to set up port forwarding to access the **system** service.
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+kubectl port-forward svc/system-service $SYSTEM_NODEPORT:9080
+```
+{: codeblock}
+
+Then, open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Run the following commands to set up port forwarding to access the **inventory** service.
+```
+INVENTORY_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services inventory-service`
+kubectl port-forward svc/inventory-service $INVENTORY_NODEPORT:9080
+```
+{: codeblock}
+
+Then use the following commands to access your **system** microservice.
+The `-u` option is used to pass in the username `bob` and the password `bobpwd`.
+```
+SYSTEM_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services system-service`
+curl -s http://localhost:$SYSTEM_NODEPORT/system/properties -u bob:bobpwd | jq
+```
+{: codeblock}
+
+Use the following commands to access your **inventory** microservice.
+```
+INVENTORY_NODEPORT=`kubectl get -o jsonpath="{.spec.ports[0].nodePort}" services inventory-service`
+curl -s http://localhost:$INVENTORY_NODEPORT/inventory/systems/system-service | jq
+```
+{: codeblock}
+
+When you're done trying out the microservices, press **CTRL+C** in the command line sessions
```

Comment:
what is the goal/result of pressing ctrl+C? does it close the session? Should give the user some indication of why so they can anticipate a successful response

Reply:
unless this is explained earlier in the guide..